mixins/form.jsx

```diff
@@ -96,7 +95,7 @@ module.exports = {
     this.updateHeight();
   },
   submit: function(action, props, callback) {
-    props.localeCode = this.state.localeCode || "US";
+    props.locale = this.props.locales[0];
```

Comment:
Yeah, I think we need something like that, and only do it for paypal.

Yes, I need it for url changes, paypal is the fringe case that only happens in one spot.